lib/Rex/Virtualization/LibVirt/create.pm

```diff
@@ -83,7 +83,7 @@ sub execute {
       my $size = $_->{'size'};
       if ( !is_file( $_->{"file"} ) ) {
         Rex::Logger::debug("creating storage disk: \"$_->{file}\"");
-        i_run "$QEMU_IMG create -f $_->{driver_type} $_->{'file'} $size",
+        i_run "$QEMU_IMG create -f $_->{driver_type} '$_->{'file'}' $size",
```

Comment:
It feels like constructs like `'$_->{'file'}'` are starting to be a bit too many single quotes close to each other, making it hard to read what's going on. Perhaps these cases can be simplified by removing the single quotes inside the curly braces and/or using `qq()` around the string?

I think it can be fixed in a followup commit during a manual merge.